### PR TITLE
COMMERCE-7818 Commerce order item decimal quantity UI and Formatter

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	enabled = false, immediate = true,
+	service = CommerceOrderItemQuantityFormatter.class
+)
+public class CommerceOrderItemQuantityFormatterImpl
+	implements CommerceOrderItemQuantityFormatter {
+
+	@Override
+	public String format(
+		CommerceOrderItem commerceOrderItem,
+		CPMeasurementUnit cpMeasurementUnit, Locale locale) {
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java:53]: CPMeasurementUnit can be retrieved from the CommerceOrderItem, but I see you need something to overload the other method

**Review by luca-pellizzon — APPROVED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceOrderItemQuantityFormatterImpl.java:53]: I can remove it and retrieve it from CommerceOrderItem, you're right. Thanks

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**